README.md

```diff
@@ -1,23 +1,51 @@
-# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
-
-## Overview
+# Beetbox - Drupal development on Vagrant
 
 Beetbox is a Vagrant configuration that provides your team with a versatile development environment for Drupal projects using Virtualbox for VMs (virtual machines).
 
-## Why use Beetbox?
+The project has a defined set of default configuration attributes so it will work without any custom configuration. All customisation comes in the form of overrides as undefined attributes will revert to the default.
+
+[![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox) [![Documentation Status](https://readthedocs.org/projects/beetbox/badge/?version=latest)](http://beetbox.readthedocs.org/en/latest/?badge=latest)
+
+
+## Features
+
+* Supports for different VMs per branch.
+* Takes advantage of Vagrant 1.8’s snapshots to manage VMs for speed and disk efficiency.
+* Uses ansible for provision but it's not required on the host machine.
```

Comment:
Partial bad copy paste of a suggested edit. Will make the changes now and force push.